ray-operator/controllers/ray/common/constant.go

```diff
@@ -103,6 +103,9 @@ const (
 
 	// Default autoscaler image when running Ray at versions older than 2.0.0
 	FallbackDefaultAutoscalerImage = "rayproject/ray:2.0.0"
+
+	// Finalizers for RayJob
+	RayJobStopJobFinalizer = "ray.io/rayjob-stopjob-finalizer"
```

Comment:
I think just `ray.io/rayjob` would be more idiomatic.

Reply:
I thought maybe it is possible to have multiple finalizers for a RayJob. For example, we need to: (1) stop the job and (2) persist some data before deleting the CR, and thus we need 2 finalizers. I am not familiar with the pattern of finalizers, so this is my guess.

Reply:
I think usually you aim to have one finalizer per controller that's cleaning up the resource.
Once all of the cleanup is done, you remove the finalizer.

Reply:
We may have multiple steps to stop the job, and each step can provide different semantics, e.g. best effort, at-least-once, at-most-once, and so on. Using one finalizer to control all of them might cause some questions. However, many finalizers will also increase the complexity of KubeRay. Hence, I think it makes sense to assume only 1 finalizer at this moment. I will update the name to `ray.io/rayjob-finalizer`.

Reply:
Updated

Reply:
We can always update this later -- the finalizer's name isn't part of a public api.